lib/txutils.js

```diff
@@ -15,11 +17,25 @@ function add0x (input) {
   }
 }
 
+function strip0x (input) {
+  if (typeof(input) !== 'string') {
```

Comment:
You can just use the `if` statement here, it makes it easier to read.
And if someone calls this function passing something else than a `string` you should throw an Error.
For ex:
```
if (typeof(input) !== 'string') {
      throw new Error('Parameter is not a string');
}
if (input.length >= 2 && input.slice(0,2) === '0x') {
     return input.slice(2);
}

return input;
```

Reply:
Good point, I’ll add this update as well.